xml/System/Half.xml

```diff
@@ -889,8 +961,8 @@
         <ReturnType>System.Half</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Represents positive infinity. This field is constant.</summary>
```

Comment:
```suggestion
        <summary>Represents positive infinity.</summary>
```

Reply:
Makes sense to not say "field", since it's a property. But isn't its value constant? Should we say instead "The value is constant"?